client/src/components/Search.js

```diff
@@ -3,35 +3,33 @@ import { useState } from "react";
 
 const Search = ({ setSearchQuery }) => {
 	const [searchInput, setSearchInput] = useState("");
-
+	const [isLoading, setIsLoading] = useState(false);
 	const handleChange = (event) => {
 		setSearchInput(event.target.value.trim());
 	};
 
 	const handleSubmit = async (event) => {
 		event.preventDefault();
+		setIsLoading(true);
 		setSearchQuery(searchInput);
+		setTimeout(() => {
+			setIsLoading(false);
+ }, 3000);
 	};
 
 	return (
-		// <div className="search-section">
-		// 	<form onSubmit={handleSubmit}>
-		// 		<div className="search-container">
-		// 			<input
-		// 				type="text"
-		// 				className="search-input-field"
-		// 				placeholder="Search"
-		// 				onChange={handleChange}
-		// 			/>
-		// 		</div>
-		// 	</form>
-		// </div>
-
 		<form className="search-box" onSubmit={handleSubmit}>
         <input type="text" name="" id="" placeholder="search..." onChange={handleChange} />
-        <a href="##" className="icon">
+        {/*<a href="##" className="icon">
             <i className="fas fa-search"></i>
-        </a>
+	</a>*/}
+		<a href="##" className="icon">
+		{isLoading ? (
+<i className="fas fa-spinner fa-spin"></i>
+		) : (
+<i className="fas fa-search"></i>
+		)}
+ </a>
```

Comment:
This is a great idea! Replacing the search icon with a spinner icon really makes sense - nice work!